src/game.cpp

```diff
@@ -1518,6 +1518,9 @@ bool game::do_turn()
     // Process power and fuel consumption for all vehicles, including off-map ones.
     // m.vehmove used to do this, but now it only give them moves instead.
     for( auto &elem : MAPBUFFER ) {
+        if( sm->vehicles.empty() ) {
```

Comment:
`sm` is only initialized three lines below.

Reply:
Just as well I have a better fix ready.